src/snowflake/snowpark/functions.py

```diff
@@ -11031,6 +11031,7 @@ def make_interval(
 def interval_year_month_from_parts(
```

Comment:
can you mention the change in the changelog that we no longer alias the column name